semgrep/semgrep/core_output.py

```diff
@@ -122,14 +123,18 @@ def parse(cls, raw_json: JsonObject) -> "CoreError":
             level_str = "WARN"
         level = Level[level_str.upper()]
 
+        details = raw_json.get("details")
```

Comment:
Wait what does this do if details isn't there?

Reply:
Returns None

Reply:
So https://github.com/returntocorp/semgrep/pull/3876#diff-ac75fcfbab642d994ff6593c78b023777d5cd0bb672397b73bbe950f96e26bc6R145 will be used